plans/install_published.pp

```diff
@@ -195,6 +200,16 @@
       |- EOW
     }
     set_var($t, 'app_hostname', $configured_hostname)
+    $configured_wh_hostname = pick($config_hash.dig('spec', 'values', 'backend_hostname', 'value'), $configured_hostname)
+    set_var($t, 'webhook_hostname', $configured_wh_hostname)
+    $webhook_port = $configured_wh_hostname != $configured_hostname ? {
+      true  => '443',
+      false => pick(
+        $config_hash.dig('spec', 'values', 'backend_port', 'value'),
+        '8000',
```

Comment:
Does it matter that the spec will not have backend_port set?

Reply:
There's a default. I suppose we could provide an explicit value, rather than hardcode this to match the default in holodeck-manifests.